templates/digital-asset-template/scripts/move/upgrade.js

```diff
@@ -0,0 +1,35 @@
+require("dotenv").config();
+const cli = require("@aptos-labs/ts-sdk/dist/common/cli/index.js");
+
+async function publish() {
+  if (!process.env.VITE_COLLECTION_CREATOR_ADDRESS) {
+    throw new Error("VITE_COLLECTION_CREATOR_ADDRESS variable is not set, make sure you set it on the .env file");
+  }
+
+  if (!process.env.VITE_MODULE_ADDRESS) {
+    throw new Error(
+      "VITE_MODULE_ADDRESS variable is not set, make sure you have published the module before upgrading it",
+    );
+  }
+
+  const move = new cli.Move();
+
+  move
+    .upgradeObjectPackage({
+      packageDirectoryPath: "move",
+      objectAddress: process.env.VITE_MODULE_ADDRESS,
+      namedAddresses: {
+        // Upgrade module from an object
+        launchpad_addr: process.env.VITE_MODULE_ADDRESS,
+        // This is the address you want to use to create collection with, e.g. an address in Petra so you can create collection in UI using Petra
+        initial_creator_addr: process.env.VITE_COLLECTION_CREATOR_ADDRESS,
+        // Our contract depends on the token-minter contract to provide some common functionalities like managing refs and mint stages
+        // You can read the source code of it here: https://github.com/aptos-labs/token-minter/
+        // Please find it on the network you are using, This is testnet deployment
+        minter: "0x3c41ff6b5845e0094e19888cba63773591be9de59cafa9e582386f6af15dd490",
+      },
+      profile: process.env.VITE_APP_NETWORK,
+    })
+    .then(console.log);
```

Comment:
why we need to console log here?

Reply:
just to display the result

Reply:
dont think we need it? it outputs any cli response to the console already

Reply:
remvoed